doc/design/vic-machine.md

```diff
@@ -63,6 +63,111 @@ To create a VCH, disambiguation of which resource to use from the available set
+* -volume-store - datastore prefixes under which volumes can be created. These prefixes are mapped to labels that can be reference via the --opts mechanism when calling `docker volume create`
+* -docker-network - vSphere networks that should be exposed via `docker network` commands. As with the datastores this is a mapping from vSphere name to docker name. Ideally the docker name should express something about the purpose of presenting the network such as `internet`, `intranet`, or `databases`
+* -docker-network-ipam - this is an optional argument furnishing additional information for controlling IP address management on the network, in the form `ipaddress-range/mask,gateway`. If not specified, DHCP is used.
+* -container - this allows containers to address a specific FQDN as if it were itself a container managed by the VCH, specified in the form of `FQDN:alias.network`.  
+
+
+### Manifest file - create
+Note that the various resource paths end in a `/` - this signifies that it's acceptable for additional refinement to be provided when using this manifest file:
+
+```
+vic-machine create
+-TO=file::///home/joe/vch/edw.manifest
+-target=vcenter
+-user=root
+-password=****
+-compute-resource=/example-datacenter/host/mycluster/Resources/edw-pool/
+-image-store=ds://vsan-datastore/containers/edw
+-container-store=ds://vsan-datastore/edw/edw/
+-client-network="VM Network"
+-management-network="management-net"
+-volume-store=ds://edw-san/reports/:reports
+-volume-store=ds://edw-san/data/:data
+-docker-network=private-vlan12d7f2:bridge
+-docker-network=edw-corp-net:backend
+-docker-network-ipam=edw-corp-net:10.118.78.128-192/24,10.118.78.1
+-docker-network=internet-corp-net:frontend
+-container=oracledb.edw.corp.net:db.backend
+```
+
+### Manifest file - use
+When using a manifest file it's necessary to fully qualify resources that are left open (i.e. trailing `/` in the manifest). In the case of mappings where it is `source/:destination`, the destination label is used as the resource reference; this avoids a requirement to know the manifest path, although there is no intent at this time to obfuscate specifications beyond encrypting credentials and target URI.
+
+```
+vic-machine create
+-FROM=file::///home/joe/vch/edw.manifest
+-compute-resource=webteam
+-container-store=web
+-volume-store=reports/web:report
+-docker-network=bridge
+-name=web-team-private
+```
+
+The exact consumption mechanic of manifests is still quite uncertain. The example above is working on the premise that all mappings in the manifest are optional and are bound only if referenced, whether simply by label or as part of a refinement. The example above does not reference the _frontend_ network, so no containers can be attached to it.
+
+### List existing VCHs
+List the existing VCHs under the specified compute resource. The example below will list all VCHs in the target vCenter.
```

Comment:
Still thinking through this. My inclination is towards basic informational listing:
```
MBUsed MHzUsed adminEmail /inventory/path/to/vch
```

This should probably go to UX to find out what viadmin's actually want to know about systems in this fashion.